recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "2.7.4" %}
+{% set version = "2.11" %}
 
 package:
   name: pyicu
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/P/PyICU/PyICU-{{ version }}.tar.gz
-  sha256: c0655302e2aea16f9acefe04152f74e5d7d70542e9e15c89ee8d763c8e097f56
+  sha256: 3ab531264cfe9132b3d2ac5d708da9a4649d25f6e6813730ac88cf040a08a844
 
 build:
-  number: 0
+  number: 1
```

Comment:
if this is a package update (from older to newer version) build number should be reset to 0.

Reply:
true, this started as a rebuild, but ended up as an upgrade and I forgot to revisit build number